docs/install/manage-add-ons-packages.md

```diff
@@ -0,0 +1,407 @@
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+Else click the {guilabel}`Install` button to complete installation of the add-on.
+
+
+(manage-check-out-an-add-on)=
+
+### Check out an add-on
+
+Add the add-on in {file}`requirements.txt`:
+
+```
+collective.easyform
+```
+
+In {file}`mx.ini`, specify the information to check out the add-on:
+
+```ini
+[collective.easyform]
+url=git@github.com:collective/collective.easyform.git
+branch=dev-branch-name
+extras=test
+```
+
+Add it to {file}`instance.yml` to let Zope know that this add-on should be loaded:
+
+```yaml
+default_context:
+    load_zcml:
+        package_includes: [
+            'project_title',
+            'collective.easyform',
+        ]
+```
+
+Stop the backend with {kbd}`ctrl-c`.
+Then apply your changes and start the backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+Else click the {guilabel}`Install` button to complete installation of the add-on.
+
+
+(manage-pin-the-version-of-a-plone-package-against-constraints-label)=
+
+### Pin the version of a Plone package against constraints
+
+A version can **not** be pinned in `constraints.txt` when the package is mentioned in the constraints of Plone.
+Any other package version could be pinned in `constraints.txt`.
+ 
+Pin the version of a Plone package in {file}`mx.ini`:
+
+```ini
+[settings]
+# constraints of Plone packages
+version-overrides =
+    plone.api>=2.0.0a3
+```
+
+Apply your changes and restart backend:
+
+```shell
+make build-backend
+make start-backend
+```
+
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+
+
+(manage-checkout-a-plone-package-label)=
+
+### Check out a Plone package
+
+This section covers how to check out a Plone Core package for development.
+
+Add the Plone package you want to check out in {file}`mx.ini`.
+
+```ini
+[plone.restapi]
+url = git@github.com:plone/plone.restapi.git
+branch = master
+extras = test
+```
+
+Stop the backend with {kbd}`ctrl-c`.
+Then apply your changes and start the backend.
+
+```shell
+make build-backend
+make start-backend
+```
+
+In your web browser, and assuming you are currently logged in as `admin`, visit the URL http://localhost:8080/Plone/prefs_install_products_form.
+An upgrade step might need to be performed in the Plone control panel.
+Follow the upgrade information, if present.
+
+
+(manage-build-and-start-your-instance-label)=
+
+### Build and start your instance
+
+Build and run the Plone backend.
```

Comment:
Sorry, just one more.
A build is necessary on changes like mentioned in sections above (another add-on needed, etc) So maybe one sentence like "You made changes, cause you need another add-on, you forced to override a Plone core package, etc.. Then a build and restart is needed." would illustrate the need for a build and restart .

Reply:
Good point. Added explanation for why it is needed in ce35ee48ed10078f3628ee91e2f9ab5e04d96a69